### ISSUE TITLE
Accessibility issues with Registration Page

### ISSUE THREAD

**Issue by mgifford** [strapi/strapi#8675]
## Bug report

Starting with strapi-starter-gatsby-blog-v2 I ran into a number of accessibility problems with the registration page - http://localhost:1337/admin/auth/register-admin

The Wave Toolbar & Accessibility Insights both highlighted issues.  I've attached a screenshot of the former and a dump of the HTML in the latter.

![Results of WAVE review of registration page](https://user-images.githubusercontent.com/116832/99412803-1e7a5700-28c3-11eb-8fcf-f5bc2ba2dcaf.png)

[AutomatedChecks_20201117_Registeradmin.html.zip](https://github.com/strapi/strapi/files/5554583/AutomatedChecks_20201117_Registeradmin.html.zip)

### Describe the bug

Basic accessibility issues may be blocking people from disabilities from accessing your editor. Editing tools should be aspiring to meet https://www.w3.org/TR/ATAG20/ so that it is easier for editors to create accessible content. They should also be accessible to not restrict access.

### Steps to reproduce the behavior

1. Go to http://localhost:1337/admin/auth/register-admin
2. After installing https://wave.webaim.org/extension/ and/or https://accessibilityinsights.io/ click on the extension button in the browser toolbar
3. With Accessibility Insights, you'll need to click on FastPass to see what errors are caught with simple automated testing.

### Expected behavior

There shouldn't be simple accessibility errors in source projects like this that are used by thousands of other users. 

### Screenshots

See above

**Comment by derrickmehaffy** [comment]
Hi @mgifford thank you for the bug report, currently we don't have great accessibility.

I'm going to set this as a duplicate of https://github.com/strapi/strapi/issues/8649 but thank you for the additional information. It may take us quite a bit of time to properly add in some accessibility options as we have a very full plate over the next few months.

**Comment by mgifford** [comment]
@derrickmehaffy I'm more concerned about what your team can do to demonstrate that you are interested improving the accessibility of your code base. As with security, there are always going to be issues and it can always be better.  

Can the strapi team try to run tools like WebAim's WAVE Toolbar on the pages of the next release to start to catch some of the low hanging fruit? Could there be an accessibility tag to track these issues rather than closing them?

**Comment by derrickmehaffy** [comment]
> I'm more concerned about what your team can do to demonstrate that you are interested improving the accessibility of your code base. As with security, there are always going to be issues and it can always be better.

We are certainly interested, but we have a limited number of team members and quite a lot to focus on right now

> Can the strapi team try to run tools like WebAim's WAVE Toolbar on the pages of the next release to start to catch some of the low hanging fruit? Could there be an accessibility tag to track these issues rather than closing them?

I can't make that decision for the project itself, but I have passed the information along to the teams that add in testing.

---

This bug report was closed and marked as a duplicate to your feature request, because at the moment yes we do have poor accessibility. Our process of handling new requests like these take time and unfortunately we can't just immediately divert resources to working on it right away. Typically we make plans for at least the next 2 quarters in advance that will generally fill our team's capacity. Naturally we do leave a little wiggle room for tackling smaller bugs and some of the critical security issues.

Our product team reviews each feature request and will provide direct feedback if it's something we can include in our plans.

You can find our public roadmap here:  

https://portal.productboard.com/strapi/1-roadmap/tabs/3-planned
https://portal.productboard.com/strapi/1-roadmap/tabs/6-in-progress-design
https://portal.productboard.com/strapi/1-roadmap/tabs/5-in-progress-development

---

As a foot note to the above, you are also more than welcome to submit some pull requests if you feel you can contribute to making accessibility better in Strapi, as I'm sure you are aware we are open source. This simply means we greatly appreciate our community and anything they can provide.

**Comment by mgifford** [comment]
This is an excellent response. Thanks @derrickmehaffy - happy to hear that you are open to PR's.
